app/components/collections/work_row_component.html.erb

```diff
@@ -7,7 +7,7 @@
   <td><%= work.owner.sunetid %></td>
   <td><%= render Works::StateDisplayComponent.new(work_version: work_version) %></td>
   <td><%= render LocalTimeComponent.new(datetime: work.updated_at, show_time: false) %></td>
-  <td><%= attached_files.count %></td>
+  <td><%= attached_files.size %></td>
```

Comment:
This is based on an article I found about the performance of `count` vs. `size`: https://www.speedshop.co/2019/01/10/three-activerecord-mistakes.html. This can be confirmed locally by creating a work with a lot of files (I tested with 25K) and manually performing the `count` query, you will see a slow query of all objects in order to get the count of records vs. `size` not performing a query against all records.